test/mciproject.yml

```diff
@@ -80,7 +81,13 @@ tasks:
                 install_name_tool -change /usr/local/lib/libwiredtiger-$WT_VERSION.dylib $(pwd)/.libs/libwiredtiger-$WT_VERSION.dylib .libs/wt
               fi
 
-              ${test_env_vars|} TESTUTIL_ENABLE_LONG_TESTS=1 ${make_command|make} VERBOSE=1 check 2>&1
+              # Exclude csuite test which is covered by a separate task
+              dirs=$(cat build_posix/Make.subdirs | grep -v ^# | grep -v ^$ | sed '/\./d' | cut -d' ' -f1 | grep -v test/csuite)
+              echo ${dirs}
+              for dir in ${dirs}
+              do
+                ${test_env_vars|} ${make_command|make} VERBOSE=1 check -C ${dir} ${smp_command|} 
```

Comment:
Do we really want to disable the long test option (i.e. `TESTUTIL_ENABLE_LONG_TESTS=1`) for the Ubuntu build variant? There's tradeoff here we need to decide between testing scope and duration. There were over 20mins saving if disabling the long test option (see comparison table [here](https://jira.mongodb.org/browse/BUILD-4395?focusedCommentId=2042941&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2042941)).

Reply:
I'd like to run the long tests individually as separate jobs. That is going to require work as more tests are added, but I think it's worthwhile for the benefit of parallelizing our testing.

Reply:
When talking about `separate jobs`, do you mean `separate build variants` in evergreen?

Reply:
I meant separate `tasks`